src/vss_tools/exporters/binary.py

```diff
@@ -121,22 +123,32 @@ def cli(
     overlays: tuple[Path],
     quantities: tuple[Path],
     units: tuple[Path],
+    types: tuple[Path],
+    types_output: Path | None,
 ):
     """
     Export to Binary.
     """
 
-    tree, _ = get_trees(
+    tree, datatype_tree = get_trees(
         vspec=vspec,
         include_dirs=include_dirs,
         aborts=aborts,
         strict=strict,
-        extended_attributes=extended_attributes,
         quantities=quantities,
         units=units,
+        types=types,
         overlays=overlays,
     )
+
     log.info("Generating binary output...")
+    if datatype_tree:
+        if types_output:
+            with open(str(types_output), "wb") as f:
+                export_node(datatype_tree, f)
+            log.info("Binary datatype tree output generated in %s", types_output)
+
     with open(str(output), "wb") as f:
         export_node(tree, f)
-    log.info("Binary output generated in %s", output)
+    log.info("Binary main tree output generated in %s", output)
+
```

Comment:
pre-commit complains on this empty line (i.e. two end of line at end of file, should only be one.)

Reply:
@UlfBj - if you just remove a line break here then two of the CI-failures should dissappear